bases/br_ibge_populacao/brasil/table_config.yaml

```diff
@@ -117,16 +103,53 @@ partitions: # REQUIRED
 # Algumas colunas existirão apenas na tabela final, você as construirá em `publish.sql`.
 # Para esses, defina is_in_staging como False.
 # Além disso, você deve adicionar as colunas de partição aqui e definir is_partition como True.
-columns: # REQUIRED
-  
-    -   
-        name: ano
-        description: Ano
-        is_in_staging: True # Bool [True, False], whether the column is in the staging table
-        is_partition: False # Bool [True, False], whether the column is a partition.
-  
-    -   
-        name: populacao
-        description: População
-        is_in_staging: True # Bool [True, False], whether the column is in the staging table
-        is_partition: False # Bool [True, False], whether the column is a partition.
+columns:
+    - name: ano
+      bigquery_type: int64
+      description: Ano
+      temporal_coverage:
+          - (1)
+      covered_by_dictionary: no
+      directory_column:
+          dataset_id: br_bd_diretorios_data_tempo
+          table_id: ano
+          column_name: ano
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging: true
+      is_partition: false
+    - name: populacao
+      bigquery_type: int64
+      description: População total
+      temporal_coverage:
+          - (1)
+      covered_by_dictionary: no
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit: person
+      has_sensitive_data: no
+      observations:
+      is_in_staging: true
+      is_partition: false
+    - name: populacao_economicamente_ativa
+      bigquery_type: int64
+      description: População Economicamente Ativa
```

Comment:
descreve melhor isso aqui pq se nao o usuário tem que ir procurar

Reply:
qual conceito vc utilizou? Como o ibge define PEA?

Reply:
O de "Pessoas de 14 anos ou mais de idade, por condição em relação à força de trabalho e condição de ocupação", melhor acrescentar em observação ou na descrição?

Reply:
descrição